test/integration/fixtures/blocks/core__button__deprecated-v12.html

```diff
@@ -0,0 +1,17 @@
+<!-- wp:buttons {"align":"wide","layout":{"type":"flex","justifyContent":"center"}} -->
+<div class="wp-block-buttons alignwide"><!-- wp:button {"fontSize":"xx-large"} -->
```

Comment:
The name of this fixture indicates that it targets the `Button` block. The e2e error is due to the top-level block in this fixture being a `Buttons` block. See the full content test for [implementation details](https://github.com/WordPress/gutenberg/blob/trunk/test/integration/full-content/full-content.test.js#L240) of this check.

The best bet here is likely just to make the feature a single Button block with all the typography styles and classes on the wrapper. The easiest way to do this is just checkout trunk and within the editor assign all the various typography styles, then switch to the code editor and copy the block's markup.